src/server/game/Entities/Unit/Unit.h

```diff
@@ -1763,6 +1763,14 @@ class TC_GAME_API Unit : public WorldObject
         void UpdateSplineMovement(uint32 t_diff);
         void UpdateSplinePosition();
 
+        void InterruptMovementBasedAuras();
+
+        struct
+        {
+            bool Relocated = false;
+            bool Turned = false;
+        } m_positionUpdateInfo;
```

Comment:
C-style bby
better move it to the variable declaration block 😄

Reply:
and maybe give the structure a cool name too

Reply:
and even maybe transform into a class? dunnow, these days choosing between class and struct seems more like a preference to me... rather than an actual difference

Reply:
and even maybe drop the `m_` naming style 💃

Reply:
Well, as for the struct, I just did not feel like giving it a name because this place here is the only location where it's actually used so I thought it would be the best to just put it in there right away instead of making it a global thing